signingscript/src/signingscript/task.py

```diff
@@ -55,6 +61,9 @@
         "autograph_rsa": sign_file_detached,
         "apple_notarization": apple_notarize,
         "apple_notarization_geckodriver": apple_notarize_geckodriver,
+        # This format is handled in script.py
+        # "apple_notarization_stacked": apple_notarize_stacked,
+        "apple_notarization_stacked": noop_sign,
```

Comment:
This is pretty hacky. My suggestion for an alternative would be to add a layer of indirection: have all of the files passed to _all_ of these handlers, and then have them deal with iterating over the files. This would clean-up the `script.py` changes a lot as well.